### PR TITLE
Enable --yk-insert-stackmaps and fix fallout.

### PR DESCRIPTION
**Pull request by vext01**
This is companion to https://github.com/ykjit/ykllvm/pull/38.

Compiler change merges first.

### CODE REVIEW

File: yksgi/src/lib.rs [hunk 1]
```diff
@@ -13,7 +13,8 @@ use std::slice;
 mod llvmbridge;
 use llvmbridge::{get_aot_original, BasicBlock, Module, Type, Value};
 
-static ALWAYS_SKIP_FUNCS: [&str; 6] = [
+static ALWAYS_SKIP_FUNCS: [&str; 7] = [
```
Comment (ltratt) [inline yksgi/src/lib.rs:16]: I wonder if this can be `[&str; _]` or do we have to hard-code the constant?

Comment (vext01) [reply yksgi/src/lib.rs:16]: I wish we could, but alas:

```
16 | static ALWAYS_SKIP_FUNCS: [&str; _] = [
   |                                  ^ `_` not allowed here
```

Also tried omitting it entirely, but that's also invalid:

```
16 | static ALWAYS_SKIP_FUNCS: [&str] = [
   |                           ^^^^^^ doesn't have a size known at compile-time
   |
```

It irritates me too.

**Comment by vext01** [comment]
This is now ready to go.

**Comment by vext01** [comment]
```
thread '<unnamed>' panicked at 'called `Option::unwrap()` on a `None` value', yktrace/src/hwt/mapper.rs:250:33
```

investigating.

**Comment by vext01** [comment]
bors try

**Comment by ltratt** [comment]
I guess this doesn't need squashing and is ready for merging as-is?

**Comment by vext01** [comment]
OK for me to do so?

**Comment by ltratt** [comment]
As you wish.

**Comment by vext01** [comment]
splat.

**Pull request merged**